bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -10,7 +10,6 @@ use core::cmp::{Ordering, PartialOrd};
 use core::{fmt, mem};
 
 use internals::write_err;
-use io::{Read, Write};
```

Comment:
I'd do the opposite: `use` the traits and remove `io::` everywhere.

Reply:
Can do.

Reply:
Done in #2262, you asked for it, now you get to review it :)